src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-accordion.html.twig

```diff
@@ -0,0 +1,75 @@
+{#
+  Parameters:
+  - "items" (array of objects)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+
+{% set message = '' %}
+{% set component = 'accordion' %}
+
+{% if items is defined and items is not empty and items is iterable %}
+  {% for item in items %}
+    {% if item.id is not defined or item.id is empty %}
+      {% set no_id_case = true %}
+    {% endif %}
+    {% if item.content is not defined or item.content is empty %}
+      {% set no_content_case = true %}
+    {% endif %}
+    {% if item.toggle is not defined or item.toggle is empty %}
+      {% set no_toggle_case = true %}
+    {% else %}
+      {% if item.toggle.label is not defined or item.toggle.label is empty %}
+        {% set no_toggle_label_case = true %}
+      {% endif %}
+      {% if item.toggle.icon is not defined or item.toggle.icon is empty %}
+        {% set no_toggle_icon_case = true %}
+      {% else %}
+        {% if item.toggle.icon.name is not defined or item.toggle.icon.name is empty %}
+          {% set no_toggle_icon_name_case = true %}
+        {% endif %}
+        {% if item.toggle.icon.path is not defined or item.toggle.icon.path is empty %}
+          {% set no_toggle_icon_path_case = true %}
+        {% endif %}
+      {% endif %}
+    {% endif %}
+  {% endfor %}
+{% else %}
+  {% set no_items_case = true %}
+{% endif %}
+
+{% set not_compliant =
+  no_items_case or
+  no_id_case or
+  no_content_case or
+  no_toggle_case or
+  no_toggle_label_case or
+  no_toggle_icon_case or
+  no_toggle_icon_label_case or
+  no_toggle_icon_path_case
+%}
+
+{% if no_items_case %}
+  {% set message = message ~ ' It looks like we completely miss the "items" here.' %}
+{% endif %}
+{% if no_id_case %}
+  {% set message = message ~ 'The "id" is needed in each element to make the accordion work.' %}
```

Comment:
I think a space is missing at beginning of the string, should it be?
`{% set message = message ~ ' The...`
Maybe same for next ones?

Reply:
yep ;)